addons/web/static/tests/views/fields/text_field_tests.js

```diff
@@ -633,4 +645,36 @@ QUnit.module("Fields", (hooks) => {
         await nextTick();
         assert.containsOnce(document.body, ".o_popover .o_model_field_selector_popover");
     });
+
+    QUnit.test("text field should vertical autoresize when saving", async function (assert) {
+        serverData.models.partner.fields.foo.type = "text";
+        serverData.models.partner.records[0].foo = "1"
+        await makeView({
+            type: "form",
+            resModel: "partner_list",
+            resId: 1,
+            serverData,
+            arch: `
+                <form>
+                    <field name="partner_ids" widget="section_and_note_one2many">
```

Comment:
section_and_note_one2many and section_and_note_text aren't web fields, so this test won't pass/run as expected if only web is installed. I guess you can observe the same behavior with the classic one2many and text fields (you may need to set widget="text" though)